dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/cli/ProjectUtil.java

```diff
@@ -43,7 +44,7 @@ public static void createProject(String name, String path, String url, String co
 		SubMonitor mon = SubMonitor.convert(monitor, NLS.bind(Messages.CreateProjectTaskLabel, name), 100);
 		Process process = null;
 		try {
-			process = CLIUtil.runCWCTL(CLIUtil.GLOBAL_INSECURE, CREATE_CMD, new String[] {URL_OPTION, url, CLIUtil.CON_ID_OPTION, conid}, new String[] {path});
+			process = CLIUtil.runCWCTL(CLIUtil.GLOBAL_INSECURE, CLIUtil.GLOBAL_JSON, CREATE_CMD, new String[] {PATH_OPTION, path, URL_OPTION, url});
```

Comment:
Global options must all be put in a string array.  Use CLIUtil. GLOBAL_JSON_INSECURE to get an array with both options.